neard/src/cli.rs

```diff
@@ -123,27 +123,28 @@ pub(super) enum NeardSubCommand {
     /// DEPRECATED: this command has been renamed to 'localnet' and will be removed in a future
     /// release.
     // TODO(#4372): Deprecated since 1.24.  Delete it in a couple of releases in 2022.
-    #[clap(name = "testnet")]
+    #[clap(name = "testnet", hide(true))]
```

Comment:
I was under the impression that `hide` is enough, i.e. that `(true)` is not necessary. Same with `forbid_empty_values`, `arg_required_else_help` etc.

Reply:
`hide`, `forbid_empty_values`, `arg_required_else_help` are Command methods and their syntax has to be in either of the two formats:

```
method = arg
```
or

```
method(arg1, arg2)
```

https://github.com/clap-rs/clap/blob/v3.1.6/examples/derive_ref/README.md#terminology

Reply:
kk, sg, though in that case for consistency with `name` I’d go for `hide = true` myself.

Reply:
got it - will update changes to the `method = arg` format for consistency